change/@internal-react-composites-7fa0abbc-d304-4a7d-9889-bf8d1677770c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
This should be "prerelease".

Only PRs that affect the _stable build_ are patch / minor / major.